crates/sui-bridge-indexer/src/sui_datasource.rs

```diff
@@ -23,6 +23,9 @@ use tokio::task::JoinHandle;
 
 use crate::metrics::BridgeIndexerMetrics;
 
+const BACKFILL_TASK_INGESTION_READER_BATCH_SIZE: usize = 300;
```

Comment:
I assume this is the value that defines how much is going to be read in one shot?
Can you add a comment as to what this mean?
Also is this something that should be defined in the config file besides and env variable?

Reply:
this is the fetching concurrency in ingestion framework. https://github.com/MystenLabs/sui/blob/main/crates/sui-data-ingestion-core/src/reader.rs#L207
it guarantees at most N checkpoints can be fetched at the same time. A recent change downgraded the default value to 10, which significantly bottlenecked the backfill speed. In backfill tasks we should use a larger number, in live sync 10 is fine.

will add in the comment